### ISSUE TITLE
Endergenic generator stops working when world reloads

### ISSUE THREAD

**Issue by Nanawith7** [McJty/RFTools#1609]
if it allow 3 ticks as exact timing instead of a tick, it doesn't get disrupted by random tick chaos...

**Comment by josephcsible** [comment]
I feel like the endergenic generator is supposed to be difficult to get perfect.

**Comment by McJty** [comment]
Indeed. The entire idea behind the endergenic is to be a challenge to get right. There are enough easy power generators out there already

**Comment by Nanawith7** [comment]
thanks,i'll close it
to be honest i love generator itself and it's complexity(and graphical effect/generated power)
but if it become broken every time i reload the world, i can't use it
because that problem is not my own fault,
means nothing i can do there(perhaps rebuilt that will fix issue, but yeah)

**Comment by McJty** [comment]
Is it in a single chunk?

**Comment by Nanawith7** [comment]
yes. i test it on a chunk of superflat world

![2018-02-03_16 50 53](https://user-images.githubusercontent.com/6253047/35765199-94e20020-0902-11e8-8233-921f204fce74.png)
if you want to see setup(i can't proof it's not wrong, but worked once)

**Comment by meson11** [comment]
i had this problem too. normaly a replacement of the sequencer solves this problem, so i implemented a mechanism with block placers and breakers to replace the sequencers per hitting a button. so it is not easy to maintain this generator, but it is possible ;-)

**Comment by Rasputyne** [comment]
I'm having a similar problem, though I can better locate the issue: any and all sequencers, transmitters, and receivers reorient themselves to lay flat upon loading a world. This may not be the OP's actual issue, but it's similar enough in description that I figured I'd mention it here.

**Comment by josephcsible** [comment]
@Rasputyne that's unrelated, and it's been fixed since 7.26.
